presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/DefaultSqlCustomization.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.jdbc;
+
+import static java.util.Objects.requireNonNull;
+
+public class DefaultSqlCustomization
+        implements SqlCustomization
+{
+    private final String quoteValue;
```

Comment:
it's used to quote _names_ / _identifiers_, not _values_. 

call it `identifierQuote` as in BaseJdbcClient

Reply:
Done